autoshop/core/views.py

```diff
@@ -52,7 +54,7 @@ def register(request):
         print 'Profile for user "%s" failed to save due to validation errors: %s' % (
             request.user.username, form.errors
         )
-        return render(request, 'failure.html')
+        return Http404('Form is invalid:' + form.errors)
```

Comment:
Revert this. We will redirect the user to profile edit page with form error message in our separate story.